charts/monitoring-config/dashboards/app-requests.json

```
@@ -426,7 +429,7 @@
           },
           "editorMode": "code",
           "exemplar": true,
-          "expr": "avg(http_request_duration_seconds{namespace=\"${namespace}\", job=~\"${app}\",quantile=~\"${quantile}\"}) by (job)",
+          "expr": "sum by (job) (increase(http_request_duration_seconds_sum{namespace=\"${namespace}\", job=~\"${app}\"}[$__rate_interval])) / sum by (job) (increase(http_request_duration_seconds_count{namespace=\"${namespace}\", job=~\"${app}\"}[$__rate_interval]))",
```

`increase` is unnecessary here and obscures the intent. (`increase` is just `rate(...) * $__rate_interval` so it's a redundant factor hiding inside some unusual syntactic sugar.)

```suggestion
          "expr": "sum by (job) (rate(http_request_duration_seconds_sum{namespace=\"${namespace}\", job=~\"${app}\"}[$__rate_interval])) / sum by (job) (rate(http_request_duration_seconds_count{namespace=\"${namespace}\", job=~\"${app}\"}[$__rate_interval]))",
```

---

Haha, I went through this thought process - in my mind `rate` is just `increase(...) / $__rate_interval`, so by pretty much the exact same logic I made the opposite decision.

Happy to switch to rate though, since they're equivalent and rate is probably more idiomatic (and is fewer characters to type).